docs/resources/role.md

```diff
@@ -13,6 +13,11 @@ Role schema
 ## Example Usage
 
 ```terraform
+import {
+  id = 5
+  to = unleash_role.custom_root_role
+}
```

Comment:
5 is the id of the predefined Project Member role, maybe we should add it as a resource that is imported rather than a custom_root_role which could be misleading